app/screens/network/Network.tsx

```diff
@@ -199,7 +211,10 @@ const Network = ({ history }) => {
         <DetailsRow>
           <DetailsTextWrap>
             <DetailsText>Connected neighbors</DetailsText>
-            <Tooltip width={250} text="tooltip Connected neighbors" />
+            <Tooltip
+              width={250}
+              text="Number of connected peers. Spacemesh multiplexes messages from many subprotocols (gossip, discovery, hare, sync) over each P2P connection."
```

Comment:
Not sure we need to describe such technical details here.
Mb something like this? 🤔 
```suggestion
              text="Spacemesh syncs database and participate in the network by communicating with other connected peers"
```

@yaelmhoffman do you have any ideas about it? :)